eks/cleanup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
can we refer to these scripts to write logs, indentations, script options like `set-eou pipefail`, etc.
https://github.com/raspbernetes/k8s-gitops/blob/master/scripts/flux.sh